### DISCUSSION TITLE
Thumbnail/cover art removed as play button?

### DISCUSSION THREAD

**Discussion by TheHYPO35** [jellyfin/jellyfin-web#5616] [Q&A] [answered]
Did clicking on the thumbnail in the top-left of an episode page or cover art on the left side of a movie page get removed as a "play" button in a recent update?

Given that in the last few things I've tried to watch I've clicked on those images a few time by muscle memory, I'm pretty sure this used to exist.

If it's been removed, is there any custom CSS to bring it back?

**Comment by thornbill** [comment, marked as answer]
https://github.com/jellyfin/jellyfin-web/issues/5609#issuecomment-2132275174
